package.json

```diff
@@ -2,26 +2,20 @@
   "private": true,
   "description": "blockprotocol monorepo",
   "type": "module",
-  "workspaces": {
-    "packages": [
-      "apps/*",
-      "crates/*",
-      "packages/*",
-      "packages/@blockprotocol/*",
-      "packages/@local/*"
-    ],
-    "nohoist": [
-      "**/@types/css-font-loading-module",
-      "**/@types/prismjs"
-    ]
```

Comment:
This field is no longer supported

- https://yarnpkg.com/configuration/manifest/#workspaces
- https://yarnpkg.com/configuration/yarnrc#nmHoistingLimits